sdk/iotcentral/arm-iotcentral/README.md

```diff
@@ -1,63 +1,79 @@
+Alternatively, you can add these to the dependencies section in your package.json and then run `npm install`.
+
 ```bash
-npm install @azure/arm-iotcentral
+npm install @azure/arm-keyvault
+npm install @azure/identity
 ```
 
+Please note that while the credentials from the older `@azure/ms-rest-nodeauth` and `@azure/ms-rest-browserauth` packages are still supported, these packages are in maintainence mode receiving critical bug fixes, but no new features.
+We strongly encourage you to use the credentials from `@azure/identity` where we have new credentials and features being added.
+
 ### How to use
 
-#### nodejs - client creation and get apps as an example written in TypeScript.
+There are multiple credentials available in the `@azure/identity` package to suit your different needs. A few examples:
 
-##### Install @azure/ms-rest-nodeauth
+- credentials using the [Service Principal](https://docs.microsoft.com/cli/azure/create-an-azure-service-principal-azure-cli)
+- credentials using [Managed Identity](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview)
+- credentials for developer scenarios like Visual Studio Code or Azure CLI credentials.
+- interactive browser credential
 
-- Please install minimum version of `"@azure/ms-rest-nodeauth": "^3.0.0"`.
-```bash
-npm install @azure/ms-rest-nodeauth@"^3.0.0"
-```
+All of the above other than the Azure CLI credentials require you to register to [create an Azure App Registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals#application-registration) first.
+
+Read more in the [readme for @azure/identity package](https://www.npmjs.com/package/@azure/identity).
+
+#### nodejs - client creation and get apps as an example written in JavaScript.
+
+In node applications, we recommend using the `DefaultAzureCredential` which picks up the service principal details from your environment. If none is found, it looks for Managed Identity, Visual Studio Code credentials and Azure CLI credentials in that order.
 
 ##### Sample code
 
-While the below sample uses the interactive login, other authentication options can be found in the [README.md file of @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package
-```typescript
-const msRestNodeAuth = require("@azure/ms-rest-nodeauth");
+```javascript
+const { DefaultAzureCredential } = require("@azure/identity");
 const { IotCentralClient } = require("@azure/arm-iotcentral");
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
 
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new IotCentralClient(creds, subscriptionId);
-  const resourceGroupName = "testresourceGroupName";
-  const resourceName = "testresourceName";
-  client.apps.get(resourceGroupName, resourceName).then((result) => {
+const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
+const creds = new DefaultAzureCredential();
```

Comment:
Perhaps we can hint at the environment credentials because it is the easiest? something like this maybe?
```suggestion
// It will first try to look for the following variables in your environment: 
// `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, and `AZURE_CLIENT_SECRET`.
// If that fails, other credentials will be tried as explained before.
const creds = new DefaultAzureCredential();
```

Reply:
In https://github.com/Azure/azure-sdk-for-js/pull/13403/commits/321a0a7cf5d39590d9d03082fa4e2b89fed64bc9, I removed some of the details of the identity package in order to rely on the identity readme because these details can get stale or incomplete over time. In the same commit, I added a note on DefaultAzureCredential that should help with your observation here. Can you take a look?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/13416 is the Identity readme PR

Reply:
I like this edit, thanks!